core/src/main/java/org/apache/calcite/adapter/enumerable/EnumerableAggregateRule.java

```diff
@@ -44,7 +45,7 @@ protected EnumerableAggregateRule(Config config) {
   }
 
   @Override public @Nullable RelNode convert(RelNode rel) {
-    final LogicalAggregate agg = (LogicalAggregate) rel;
+    final Aggregate agg = (Aggregate) rel;
```

Comment:
I guess if we apply this patch, all related javadocs should be updated too.
E.g.:
`Rule to convert a {@link LogicalAggregate} to an {@link EnumerableAggregate}.`
=> should be:
`Rule to convert an {@link Aggregate} to an {@link EnumerableAggregate}.`

An similarly in the rest of the impacted classes.

Reply:
Adjusted JavaDoc as requested.